js/braziliex.js

```diff
@@ -503,6 +505,16 @@ module.exports = class braziliex extends Exchange {
         };
         return await this.privatePostCancelOrder (this.extend (request, params));
     }
+    
+    async fetchOrderStatus (id, symbol = undefined, params = {}) {
+      await this.loadMarkets ();
```

Comment:
Hi! Thx for the PR! ) One quick question: can you plz switch to 4-space indent? Otherwise it breaks the transpiler: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes. Thx again!